src/modules/python/Makefile.am

```diff
@@ -36,11 +36,11 @@
 # trademark licensing policies.
 #
 
-pbsmoduledir = $(libdir)/@PBS_PYTHON_DESTLIB@/pbs
+pbsmoduledir = $(libdir)/python/altair/pbs
```

Comment:
I removed that m4 macro only because we were substituting constant string, however `python/altair` is hardcoded at many places even if we use m4 macro, So I would like to keep my change. But if you still think we should use m4 macro I am happy to revert that change.
And if you think that we should change it everywhere and use a macro then I think we should do it in another PR.

Reply:
If it had been hardcoded everywhere in the first place, we probably would not have used a macro. I don't have a problem using the hardcoded path as it is unlikely to change.